rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,79 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.1
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka, bukka@php.net
+  * Status: Draft
+
+===== Introduction =====
+
+PHP uses voting system for deciding about changes to PHP. This was created mainly for user visible changes in language.
+It works well for that purpose but it is not that effective for deciding about purely technical changes that impact PHP
+internals and extension API's. It is not exactly clear how it should resolve conflicts (on technical bases) between
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src. Specifically its main purpose is to decide about disagreements between contributors but it is not limited to
+that only.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * contributor - Anyone who has already contributed to php-src.
+  * core developer - Anyone with write commit right to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2)
+  * php-src - Source code in https://github.com/php/php-src/.
+  * RM - PHP release managers.
+  * technical aspect - Any issue or pull request in php-src or commit to php-src master and dev branches.
+  * TC - PHP Technical Committee.
+  * user facing feature - Feature resulting in change in PHP code (e.g. new language constructs, classes, functions and so on).
+
+==== Work flow ====
+
+The work flow for TC is following:
+
+  - Any core developer can add label tc-hold on issue or pull request. 
+  - The core developer should mention the TC GitHub team to notify members of the TC.
+  - The TC discusses the issue or pull request and vote on it (this can be done privately)
+    * The TC should consult with maintainers if the issue if the problem impacting code maintained by the maintainer
```

Comment:
"if the issue if the problem". I also don't really understand this sentence.

Reply:
Sorry it was a typo, just changed the wording so hopefully it makes sense now.